### DISCUSSION TITLE
Need help with panel and api setup

### DISCUSSION THREAD

**Discussion by Danewolf** [PhlexPlexico/G5API#288] [Help]
Hello very nice tool you made for cs2

I have tried to ask several people in different communities, with no luck, i have read readme, and it's another league for me, i know basic setup with cms systems and mysql databases

How can i get it to work on my paid domain and web hosting, i have a cpanel, normally i use FTP access and upload the files and set it up through an install?

Thank you so much for this project, i really appreciate all what you do for the community

**Comment by PhlexPlexico** [comment]
Hi,

This application needs more than just cPanel. You would need access to SSH, and preferably Docker on your server. You would need to rent a server from somewhere such as Digital Ocean, Linode, Vultr, or any VPS provider. From there, you can point your URL to the server's IP address, and use docker to setup the entire application stack (SSL, reverse proxies, and web servers will be taken care of this way).

Once you have a VPS with Docker installed, you just need the one `docker-compose.yml` file, and fill in the required information. [You can use this one as an example](https://gist.github.com/PhlexPlexico/3d58b38f4b29cc540bb89be58cc702f7) as everything you need to change in it is labelled with `#CHANGEME` or something similar. I'm not really well-versed in teaching people how to use the build tools that make this application, so Google/DuckDuckGo/Bing/etc will definitely be a good tool to use to learn how to use Docker and `docker compose` and how to rent a VPS.

Thanks!

**Reply by Danewolf** [reply]
Very nice, but is it possible to get the web files? Am i able to add you on discord? It's kinda late here now, in denmark

**Reply by Danewolf** [reply]
Side note: i use Godaddy web hosting / domain

**Reply by PhlexPlexico** [reply]
As I said, this application is not just "web files". You need node.js to run on your server, which web hosting does not (normally) support. You'd need access to SSH/a terminal to the site to be able to run Linux commands to effectively "install" the application.

Sorry, but I only offer help through these platforms, as it's still public and people can use them as a reference if they themselves run into issues.

**Reply by Danewolf** [reply]
Let me look my web hosting cpanel, if i have docker option and SSH
